### PR TITLE
Add pe-source tests

### LINKED ISSUE #198: Create more pe-source pytests
# 💡 Summary #

Only a small percentage of our code is covered by tests. Possible tests:
- db connections
- data cleanup
- pdf creation

## Motivation and context ##

Improve functionality and pass Coveralls checks.

## Acceptance criteria ##

- [ ] all new and existing pytests need to pass

### PR DESCRIPTION
**Pull request by aloftus23**
fixes #198
## 🗣 Description ##
Adds pe-source tests:
1. Source name validation
2. Cybersixgill method validation
3. Credential data cleanup and function is called correctly
4. Shodan search is called correctly

## 💭 Motivation and context ##

Improve number of tests in the pe-source module.

## 🧪 Testing ##

All pytest and pre-commits passed.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by DJensen94 — APPROVED** [review]
Looks great

File: tests/test_pe_reports.py [hunk 1]
```diff
@@ -121,6 +121,10 @@ def test_reports_bad_log_level():
         assert return_code == 1, "main() should exit with error"
 
 
+# TODO: Test data cleanup in metrics for each source
+# (Credentials, Domain Masq, Shodan, Cybersixgill)
```
Comment (dav3r) [inline tests/test_pe_reports.py]: Is there a GitHub issue for the work in this TODO?  If not, please create one and then add a comment here with the issue URL.

**Review by dav3r — APPROVED** [review]
Looks good to me- I only had one question about a TODO comment.

File: tests/test_pe_reports.py (hunk 1 above)
Comment (aloftus23) [reply tests/test_pe_reports.py]: Sounds good. Created the issue and added here: https://github.com/cisagov/pe-reports/pull/201/commits/a66748c95aaa57e9298592efc3b2f0cff4505fe2

Comment (dav3r) [reply tests/test_pe_reports.py]: Perfect, thanks! 👍

**Pull request merged**